### ISSUE TITLE
Open folder UTF8

### ISSUE THREAD

**Issue by dabcorp** [barbushin/php-imap#243]
Hello,

I can not open a directory containing caratceres of the type "é"
```

$folder = mb_convert_encoding("Objets envoyés", "UTF7-IMAP", "UTF-8");
$mailboxSent = new PhpImap\Mailbox('{imap.1and1.fr:993/imap/ssl}$folder', $username, $password, __DIR__);
$mailsIds = $mailboxSent->searchMailbox('ALL');
```

Can you look at or help me?

**Comment by NickCoolii** [comment]
@dabcorp I changed here `$arg = imap_utf7_encode($arg);`
https://github.com/barbushin/php-imap/blob/master/src/PhpImap/Mailbox.php#L846-L850
with `$arg = mb_convert_encoding($arg, 'UTF7-IMAP', 'UTF-8');`.
It's works for me. Hope for you will too.

**Comment by Sebi94nbg** [comment]
@barbushin can be closed as well: [Commit a6b2dd5](https://github.com/barbushin/php-imap/pull/296/commits/a6b2dd5409ed3de62696a1705fd27fedcfda06d7)

See PR #296.
